base/setup/usetup/genlist.c

```diff
@@ -26,7 +26,7 @@
 
 /* INCLUDES *****************************************************************/
 
-#include "usetup.h"
+#include "headers/usetup.h"
```

Comment:
why introduce a headers directory?

Reply:
To clean it up a bit. It is still a work in progress, however.